src/test/integration/storage/test_db_interface_common.py

```diff
@@ -218,16 +218,6 @@ def test_get_summary(db):
     assert fw.uid not in result_sum['file exclusive sum b'], 'parent as origin but should not be'
 
 
-def test_collect_summary(db):
```

Comment:
Seeing you deleted a test and none is renamed or adapted one has to wonder if this existing functionality and the new adapted one is tested anywhere. Is it?

Reply:
`_collect_summary_for_uid_list` may not get tested but `get_summary` (which calls the function) does. I added a new test case for this function to explicitly also test summary generation for `FileObject`s